MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/snapshot/SnapshotShareActivity.java

```diff
@@ -94,39 +95,40 @@ public void onClick(View view) {
    * @param width        of map
    */
   private void startSnapShot(LatLngBounds latLngBounds, int height, int width) {
-    if (!style.isFullyLoaded()) {
```

Comment:
Shall we remove the unused private field `style`?

Reply:
Removed